godal.cpp

```diff
@@ -685,6 +685,13 @@ OGRGeometryH godal_OGR_G_Buffer(cctx *ctx, OGRGeometryH in, double tolerance, in
 	return ret;
 }
 
+int godal_OGR_G_Intersects(cctx *ctx, OGRGeometryH geom1, OGRGeometryH geom2) {
+	godalWrap(ctx);
```

Comment:
Error checking/handling should be done in the same way as in `godal_OGR_G_Buffer`

Reply:
`OGR_G_Intersects` only returns an `int`, which is `TRUE` or `FALSE`.
It does not return an handle which can be `nullptr`, so we can't have the same error handling.

Or do I miss something?